components/pkg-export-tar/src/lib.rs

```diff
@@ -0,0 +1,115 @@
+#[macro_use]
+extern crate clap;
+extern crate habitat_core as hcore;
+extern crate url;
+extern crate habitat_common as common;
+extern crate base64;
+
+extern crate hab;
+extern crate handlebars;
+
+extern crate mktemp;
+extern crate tempdir;
+extern crate failure;
+#[macro_use]
+extern crate failure_derive;
+extern crate lazy_static;
+#[macro_use]
+extern crate log;
+extern crate serde_json;
+extern crate tar;
+
+mod build;
+pub mod cli;
+mod error;
+mod rootfs;
+
+use std::path::{Path, PathBuf};
+use std::fs::File;
+pub use cli::Cli;
+pub use error::{Error, Result};
+use common::ui::UI;
+use hcore::channel;
+use hcore::url as hurl;
+use hcore::package::{PackageIdent, PackageInstall};
+use tar::Builder;
+use std::str::FromStr;
+
+pub use build::BuildSpec;
+
+/// The version of this library and program when built.
+pub const VERSION: &'static str = include_str!(concat!(env!("OUT_DIR"), "/VERSION"));
+/// The Habitat Package Identifier string for a Busybox package.
+const BUSYBOX_IDENT: &'static str = "core/busybox-static";
+
+pub fn export_for_cli_matches(ui: &mut UI, matches: &clap::ArgMatches) -> Result<()> {
+    let default_channel = channel::default();
+    let default_url = hurl::default_bldr_url();
+    let spec = BuildSpec::new_from_cli_matches(&matches, &default_channel, &default_url);
+    export(ui, spec)?;
+
+    Ok(())
+}
+
+pub fn export(ui: &mut UI, build_spec: BuildSpec) -> Result<()> {
+    let hab_pkg = build_spec.hab;
+    let build_result = build_spec.create(ui).unwrap();
+    let builder_dir_path = build_result.0.path();
+    let pkg_ident = build_result.1;
+
+    tar_command(builder_dir_path, pkg_ident, hab_pkg);
+    Ok(())
+}
+
+#[allow(unused_must_use)]
+fn tar_command(temp_dir_path: &Path, pkg_ident: PackageIdent, hab_pkg: &str) {
+    // We currently support only exporting one tar package at a time
+    // So we grab the first value from the Vec of idents
+    let pkg_ident = pkg_ident.clone();
```

Comment:
I don't think you need this line at all anymore.

Also, the comment immediately above no longer applies.